components/GothamistHomepageTopper.vue

```diff
@@ -12,12 +12,15 @@ const props = defineProps<{
 
 const featuredArticle = computed(() => props.articles[0])
 const latestArticles = computed(() => {
-  return props.articles.slice(1)
-    // remove the manually curated featured article
-    // from the list of latest articles so it
-    // doesn't show up twice in this module
-    .filter(article => article.uuid !== featuredArticle.value.uuid)
-    .slice(0,4)
+  return (
+    props.articles
+      .slice(1)
```

Comment:
We're already removing the featuredArticle from the latest articles list with the .filter below. What is this trying to do?

Reply:
ah, this is just formatting...

Reply:
Ah i see, hmm maybe i wrote that one and meant to do a slice(0) to make a copy, or maybe i didn't see it there when I wrote the bottom part.

Reply:
I mean it's redundant and should be cleaned up, but it's not causing any problems, and not really in scope for this ticket. I'm sure we'll touch loading articles or implementing other types of toppers or something like that at some point and can revisit it then.